.github/workflows/main-armel.yml

```diff
@@ -0,0 +1,33 @@
+name: Build for armel (static)
+
+on:
+  push:
+  pull_request:
+  schedule:
+    # Weekly build (on saturday)
+    - cron: '0 0 * * 6'
+
+jobs:
+  build-armel:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
```

Comment:
There is a v4 of this that I think i needed to use for something else, might as well use that one.